sandpiper/processes/wps_evaluate_rule.py

```diff
@@ -5,8 +5,8 @@
 
 from p2a_impacts.fetch_data import get_dict_val
 from p2a_impacts.evaluator import evaluate_rule
-from wps_tools.utils import log_handler, collect_args
-from wps_tools.io import log_level
+from wps_tools.logging import log_handler
+from wps_tools.io import log_level, collect_args
```

Comment:
I'm beginning to think that we may want to consider importing `wps_tools` in a more flexible manner. We have been adding so many new tools lately that we have to constantly come back up to these import statements and add specific methods. Perhaps it would be easier to just have:
```
import wps_tools.io as wps_io
log_level = wps_io.log_level
``` 
That way if add a new item, we don't have to update the import statement at all. This may not be necessary for this particular case since we are only importing 3 different things. But it may be something we should consider. Any thoughts?

Reply:
Sure, I'd say that's not really needed for this bird necessarily, but with some of the others it might be